Home.md

```diff
@@ -0,0 +1,56 @@
+![logo](https://github.com/culturegraph/metafacture-core/wiki/img/metafacture_small.png)
+
+# Metafacture Documentation
+
+Metafacture is a toolkit for processing semi-structured data with a focus on library metadata. It provides a versatile set of tools for reading, writing and transforming data. Metafacture can be used as a stand-alone application via CLI or as a Java library in other applications. There is also a playground where you can test workflows.
+
+The name Metafacture is a portmanteau of the words metadata and manufacture.
+
+Metafacture comprises three main parts: Framework, Flux and the Transformation-Module Fix. It can be extended with modules.
```

Comment:
```suggestion
Metafacture comprises three main parts: Framework, Flux and the Transformation modules Fix and Morph. It can be extended with modules.
```

Reply:
Agreed.You would mention Morph without having to go into details - as you mention modules without going into details. I propose to a) remove Fix  in "Transformation module Fix" or b) mention them both. I vote for b) also because in the Playground while you encounter the textarea named "transformationFile" there may be written "metamorph" in it  ["Morph" https://github.com/metafacture/metafacture-playground/issues/114](https://test.metafacture.org/playground/?example=Local+formeta+to+XML+%28morph%29).